scipy/stats/_stats_py.py

```diff
@@ -1529,7 +1537,8 @@ def describe(a, axis=0, ddof=1, bias=True, nan_policy='propagate'):
     if xp_size(a) == 0:
         raise ValueError("The input must not be empty.")
 
-    n = a.shape[axis]
+    n = xp.asarray(_shape_nonmasked(a, axis), dtype=xp.int64)
+    n = n[()] if n.ndim == 0 else n
```

Comment:
We can use `xp.astype` and eliminate the ternary once data-apis/array-api-compat#226 is resolved.